index.html

```diff
@@ -9,16 +9,22 @@
 </head>
 <body>
     <!-- first page -->
+    <header id="header">
+        <h1 class="logo"><a href="">My Logo</a></h1>
+        <nav class="nav_icon">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+            <a href="#"><img src="./images/menu-icon.png" alt="menu-icon"></a>
+        </nav>
+    </header>
+
     <section class="cover">
         <div class="header_margin">
-            <header id="header">
-                <h1 class="logo"><a href="">My Logo</a></h1>
-                <nav class="nav_icon">
-                    <a href="#"><img src="./images/menu-icon.png" alt="menu-icon"></a>
-                </nav>
-            </header>
             <section class="page-one" id="first_page">
-                <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see <br> you!</span></h1>
+                <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">Glad to see you!</span></h1>
```

Comment:
- [ ] I think it will be better if your headline title matches your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma in terms of color, font size and font-weight 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194621286-c651e20b-7380-4ef3-8fd1-9fba183dae54.png)  | ![image](https://user-images.githubusercontent.com/63473584/194621353-927851c3-283e-4382-a9fa-25ed2ecb1773.png)  |